homework/week1/RosmaryFC/HoroscopeHW20150508/app/src/main/java/nyc/c4q/rosmaryfc/horoscopehw20150508/MainActivity.java

```diff
@@ -0,0 +1,127 @@
+import android.os.Bundle;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBarDrawerToggle;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import java.util.Locale;
+
+
+public class MainActivity extends Activity{
+private String[] mSignTitles;
+private DrawerLayout mDrawerLayout;
+private ListView mDrawerList;
+
+private ActionBarDrawerToggle mDrawerToggle;
+private CharSequence mDrawerTitle;
+private CharSequence mTitle;
+
+        @Override
+        protected void onCreate(Bundle savedInstanceState) {
+            super.onCreate(savedInstanceState);
+            setContentView(R.layout.activity_main);
+
+            mSignTitles = getResources().getStringArray(R.array.signs_array);
+            mDrawerLayout = (DrawerLayout) findViewById(R.id.drawer_layout);
+            mDrawerList = (ListView) findViewById(R.id.left_drawer);
+
+            // Set the adapter for the list view
+            mDrawerList.setAdapter(new ArrayAdapter<String>(this,
+                    R.layout.drawer_list_item, mSignTitles));
+            // Set the list's click listener
+            mDrawerList.setOnItemClickListener(new DrawerItemClickListener());
+
+        }
+
+
+        @Override
+        public boolean onCreateOptionsMenu(Menu menu) {
+            // Inflate the menu; this adds items to the action bar if it is present.
+            getMenuInflater().inflate(R.menu.menu_main, menu);
+            return true;
+        }
+
+        @Override
+        public boolean onOptionsItemSelected(MenuItem item) {
+            // Handle action bar item clicks here. The action bar will
+            // automatically handle clicks on the Home/Up button, so long
+            // as you specify a parent activity in AndroidManifest.xml.
+            int id = item.getItemId();
+
+            //noinspection SimplifiableIfStatement
+            if (id == R.id.action_settings) {
+                return true;
+            }
+
+            return super.onOptionsItemSelected(item);
+        }
+
+/* The click listner for ListView in the navigation drawer */
+private class DrawerItemClickListener implements ListView.OnItemClickListener {
+    @Override
+    public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+        selectItem(position);
+    }
+}
+
+    private void selectItem(int position) {
+        // update the main content by replacing fragments
+        android.app.Fragment fragment = new SignFragment();
+        Bundle args = new Bundle();
+        args.putInt(SignFragment.ARG_SIGN_NUMBER, position);
+        fragment.setArguments(args);
+
+        android.app.FragmentManager fragmentManager = getFragmentManager();
```

Comment:
Your NavagationDrawerFragment class uses the support library version of fragments (android.support.v4.app.Fragment), here you are using the other version (from android.app.Fragment). 

Using the support version provides support for API's under 14, which may or may not be important to your project, but being consistent with the version you use across the project seems important: that way you won't run into problems with incompatible fragment types (the FragmentManager called here can't be used to swap support fragments)